Dockerfile

```diff
@@ -1,7 +1,9 @@
 FROM node:18.10
 
 COPY . /terra-ui/
-RUN cd /terra-ui && yarn build
+RUN cd terra-ui \
+  && PUPPETEER_SKIP_CHROMIUM_DOWNLOAD=true yarn install --immutable \
```

Comment:
Run `yarn install` to build "unplugged" packages (such as ESBuild).

Without `PUPPETEER_SKIP_CHROMIUM_DOWNLOAD =true`, Puppeteer's build failed because "The chromium binary is not available for arm64.".

Reply:
`yarn install` is currently being run from the `yarn build` script, but this PR removes it from that step.